### PR TITLE
1494

### LINKED ISSUE #1494: Improve loading screen when attempting to connect to a store or load a listing
The current loading screen is very underwhelming and also difficult to navigate (have to hit the back button in the address bar to leave)

screenshot 2018-07-30 08 48 29

Ideally we would give the user a more polished loading experience, and also provide buttons to 'cancel' or 'retry'.

screenshot 2018-08-01 11 21 07

screenshot 2018-08-01 11 22 02

➡️**Prototype here:** https://invis.io/G7NAT28FCKA#/312112604_Openbazaar-2-0-Search

### PR DESCRIPTION
**Pull request by rmisio**
This includes #1531. Once that makes it into master, I could update this one, so the diff is a little cleaner.

closes #1494

### CODE REVIEW

File: js/router.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
 import $ from 'jquery';
 import { ipcRenderer } from 'electron';
 import { Router } from 'backbone';
+import app from './app';
```
Comment (rmisio) [inline js/router.js]: Just reorganized some imports here. In general, I think it's best to keep views with views, utils with utils, models with models.

In a lot of files I'm seeing that we just dump new imports at the end as they are needed.

**Comment by rmisio** [comment]
@jjeffryes pease hold off on merging this one. I'm going to combine it with #1539 because their's a good amount of overlap in the functionality between the two.

**Comment by morebrownies** [comment]
Hey @rmisio looking good, thank you for putting this together. Here's a few small display issues I think we should try to clean up:

1) **Extra padding above avatar/store name.** Its looks like `div.padMd` has 15px padding top and bottom, but then the avatar and store name also seems to have extra padding/margin. If we could remove that, the avatar/store name would move more towards the top.

screenshot 2018-08-24 15 41 23

2) **All avatars need shadows.** It seems like all the avatars are missing shadows in the loading screen, please add them in.

screenshot 2018-08-24 15 44 32

3) **Social icons are a bit small.** Can we bump these up 15-20% or so? They're very small. 
screenshot 2018-08-24 15 45 27

4) **Failed to connect scenario missing details.** Can we make the arrow red and fade out the user's avatar when connection fails?

**Current**
screenshot 2018-08-24 15 46 22 

**Expected**
screenshot 2018-08-24 15 47 50

I think that should about do it for now. Thanks!

**Comment by rmisio** [comment]
@morebrownies 

> All avatars need shadows. It seems like all the avatars are missing shadows in the loading screen, please add them in.

The shadows are there:
screen shot 2018-08-27 at 3 28 08 pm

Maybe they're too subtle too notice? If you toggle that style, it's noticeable, but otherwise I doubt I'd notice them either.

File: js/templates/modals/verifiedModsError.html [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-    <%= ob.processingButton({
-    className: `btn clrP clrBr js-retry ${ob.fetching ? 'processing' : ''}`,
-    btnText: ob.polyT('startUp.dialogs.unableToGetVerifiedMods.btnRetry')
-    }) %>
-    <a class="btn clrP clrBr js-continue"><%= ob.polyT('startUp.dialogs.unableToGetVerifiedMods.btnClose') %></a>
-  </div>
-</div>
+<div class="contentBox clrP clrBr clrSh3">
+    <div class="padMd">
+      <div class="">
```
[the remark this answers is not in the document]
Comment (rmisio) [reply js/templates/modals/verifiedModsError.html]: I'm not sure about this one. I don't recall touching this file. Was that you @jashot7? (your modal PR is merged into this one)

File: js/templates/modals/verifiedModsError.html [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
+        <h1><%= ob.polyT('startUp.dialogs.unableToGetVerifiedMods.title') %></h1>
+      </div>
+      <div class="">
```
[the remark this answers is not in the document]
Comment (rmisio) [reply js/templates/modals/verifiedModsError.html]: @jashot7 Same question as above ^^^

File: js/languages/en_US.json [hunk 4]
```diff
@@ -347,7 +347,17 @@
       "following": "Following",
       "followers": "Followers"
     },
-    "blockedUserOverlayText": "You blocked this user"
+    "blockedUserOverlayText": "You blocked this user",
+    "loading": {
+      "connecting": "Connecting…",
+      "failedToConnect": "Failed to connect",
+      "loadingText": "Just a minute, loading %{name} from somewhere on this global, peer-to-peer network for you…",
```
[the remark this answers is not in the document]
Comment (jenncloud) [reply js/languages/en_US.json]: And not just, it went through Sam and Wolf as well! I would love to have a team on revamping the language of everything in-app and on the site to be more "flippant" like this, to have a more casual & fun voice, but alas...for now we will continue on peppering it in here and there as we see the opportunity.

**Comment by rmisio** [comment]
closing in liu of #1544

**Pull request closed without merging**